tesseract_kinematics/include/tesseract_kinematics/ikfast/ikfast_inv_kin.h

```diff
@@ -160,6 +173,9 @@ class IKFastInvKin : public InverseKinematics
   std::vector<Eigen::Index> redundancy_indices_; /**< @brief Joint indicies that have redundancy (ex. revolute) */
   std::string name_;                             /**< @brief Name of the kinematic chain */
   std::string solver_name_;                      /**< @brief Name of this solver */
+  /**< @brief combinations of free joints to sample when computing IK
+   * Example: Given 3 free joints, a valid input would be [[0,0,0][0,0,1][-1,0,1][0,2,0]] */
+  std::vector<std::vector<double>> free_joint_combos_;
```

Comment:
Being a bit pedantic, but I think the name `free_joint_states` is more clear than "free_joint_combos" since a "state" in the sense of `tesseract` implies values for various joints in a single robot pose

Reply:
That works, I was just trying to make the language as clear as possible, especially considering we will be providing a different function that takes exactly the same variable type but it represents something else.